authors/settings.py

```diff
@@ -25,7 +25,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = [
+    '*'
```

Comment:
Allowing all hosts opens the app to a lot of [attack vectors. ](https://www.divio.com/blog/django-allowed-hosts-explained/). Please make it a practice to allow only specific hosts because it's very easy to forget to update this key  later on.

Reply:
ok.